orders-core/src/test/java/io/helidon/examples/sockshop/orders/TestDefaultOrderRepository.java

```diff
@@ -0,0 +1,16 @@
+package io.helidon.examples.sockshop.orders;
+
+import javax.annotation.Priority;
+import javax.enterprise.inject.Alternative;
+import javax.interceptor.Interceptor;
+
+@Alternative
+@Priority(Interceptor.Priority.APPLICATION+10)
```

Comment:
Change priority to `APPLICATION - 5`, so it's higher than the default implementation, but lower than data store-specific implementations (Mongo, Redis, Jpa)